src/extension.ts

```diff
@@ -72,8 +72,14 @@ export function activate(context: vscode.ExtensionContext) {
   let hover = new HoverProvider();
   let disposableHover = vscode.languages.registerHoverProvider('ini', hover);
   context.subscriptions.push(disposableHover);
+
+  let disposableOneCircleTracer = vscode.commands.registerCommand('onevscode.circle-tracer', () => {
+    console.log('one circle tracer...');
+    // TODO PROCESS DIALOG
+  });
+  context.subscriptions.push(disposableOneCircleTracer);
 }
 
 export function deactivate() {
   // TODO do cleanup
-}
+}
```

Comment:
```suggestion
}

```

Please execute `infra/format` to draft in advance :)